fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1927,12 +1991,14 @@ ACTOR Future<Void> commitProxyServerCore(CommitProxyInterface proxy,
 	// retrieve TLog group ids info from ServerDBInfo
 	// TODO: change it to retrieve from master through txnStateStore
 	for (const auto& tLogSet : commitData.db->get().logSystemConfig.tLogs) {
-		if (tLogSet.isLocal) {
-			TEST(tLogSet.tLogGroups.empty()); // primary DC should have recruited ptxn TLog groups
-			commitData.tLogGroups = tLogSet.tLogGroups;
+		if (tLogSet.isLocal && !tLogSet.tLogGroupIDs.empty()) {
+			commitData.tLogGroups = tLogSet.tLogGroupIDs;
 			break;
 		}
 	}
```

Comment:
We can replace this with the data in `txnStateStore` now, since #5059 is merged. I.e., use `tLogGroupCollection` in `struct ProxyCommitData` instead.